src/js/components/search/topFilterBar/filterGroups/AgencyFilterGroup.jsx

```diff
@@ -43,6 +43,13 @@ export default class AgencyFilterGroup extends React.Component {
 
         agencies.forEach((value) => {
             let agencyTitle = value.subtier_agency.name;
+
+            if (value.agencyType === 'subtier') {
```

Comment:
Can you handle if the abbreviation is an empty string? Right now it displays as `Agency ()`.